fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergApiConverter.java

```diff
@@ -642,15 +641,19 @@ public static Map<String, String> buildViewProperties(ConnectorViewDefinition de
 
         String queryId = connectContext.getQueryId().toString();
 
-        Map<String, String> properties = com.google.common.collect.ImmutableMap.of(
-                "queryId", queryId,
-                "starrocksCatalog", catalogName,
-                "starrocksVersion", GlobalStateMgr.getCurrentState().getNodeMgr().getMySelf().getFeVersion());
+        Map<String, String> properties = new HashMap<>();
+        properties.put("queryId", queryId);
+        properties.put("starrocksCatalog", catalogName);
+        properties.put("starrocksVersion", GlobalStateMgr.getCurrentState().getNodeMgr().getMySelf().getFeVersion());
 
         if (!Strings.isNullOrEmpty(definition.getComment())) {
             properties.put(IcebergMetadata.COMMENT, definition.getComment());
         }
 
+        if (definition.getProperties() != null && !definition.getProperties().isEmpty()) {
```

Comment:
`CollectionUtils.isNotEmpty()` is preferred

Reply:
@murphyatwork any reason why it's preferred?